modules/upgrade-duration-cvo.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-openshift-upgrade-duration.adoc
+
+:_content-type: CONCEPT
+[id="cluster-version-operator_{context}"]
+= Cluster Version Operator target upgrade payload deployment
+
+The Cluster Version Operator (CVO) retrieves the target upgrade release image and applies to the cluster. All components which run as pods are updated during this phase, whereas the host components are updated by the Machine Config Operator (MCO). This process might take 45 to 120 minutes.
```

Comment:
According to the latest google doc we reviewed, here should be `60 to 120 minutes`.

Reply:
fixed

Reply:
ack